core/core.go

```diff
@@ -70,6 +71,8 @@ type Core struct {
 
 	syncTarget *types.Header
 
+	normalListBackoff uint64
```

Comment:
some code comments really would be nice :upside_down_face:

Reply:
Added the definition and also for the syncTarget

Reply:
This code is clearly self documenting, comments not needed.